imod/protocols/protocol_importSetOfTM.py

```diff
@@ -106,10 +106,13 @@ def generateTransformFileStep(self, tsId, matchBinningFactor):
         ts = self.tsDict[tsId]
         tiNum = ts.getSize()
         outputTransformFile = self.getExtraOutFile(tsId, ext=XF_EXT)
-        self.debug(f"Matching files: {self.matchingFiles}")
+        self.debug(f"Matching tsIds: {self.matchingTsIds}")
 
         for tmFilePath, _ in self.iterFiles():
-            if pwutils.removeBaseExt(tmFilePath) in self.matchingFiles:
+            tmFileBaseName = pwutils.removeBaseExt(tmFilePath)
+            # Look for basename - tsId or base -name normalized basename - tsId matches. See tomo.convert.mdoc
+            # normalizeTSId
+            if tmFileBaseName in self.matchingTsIds or normalizeTSId(tmFileBaseName) in self.matchingTsIds:
```

Comment:
isn't it the opposite, e.g. look for TSID in the filename: TS_28 in mytiltseries_TS_28_lalala ?

Reply:
Sure, it's more robust that way. I was dealing as a reference with numerical TS names to which TS_ is added. We may add the {TS} pattern to the imports to offer more versatility and make it easier for the users to make the matching when importing files that way, avoiding possible manual renaming.